extensions/wikia/ArticleVideo/scripts/recommended-video.jwplayer.js

```diff
@@ -0,0 +1,206 @@
+require([
+	'jquery',
+	'wikia.tracker',
+	'wikia.articleVideo.featuredVideo.cookies',
+	'wikia.loader',
+	'wikia.mustache'
+], function ($, tracker, featuredVideoCookieService, loader, mustache) {
+	'use strict';
+
+	var recommendedVideoSelector = '.article-recommended-video .video',
+		jwPlaylistDataUrl = 'https://cdn.jwplayer.com/v2/playlists/',
+		windowHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0),
+		$unit = $('.article-recommended-video-unit'),
+		currentVideo = null,
+		recommendedVideoElementId = 'recommendedVideo',
+		recommendedVideoData = null,
+		player = null,
+		$actualVideo = null,
+		currentItemNumber = 1,
+		isExpanded = false;
+
+	function reveal() {
+		var scrollTop = $(window).scrollTop();
+
+		if (scrollTop > windowHeight * 1.5) {
+			$unit.addClass('revealed');
```

Comment:
usually https://smacss.com/book/state is used for state changing selectors, so `is-revealed`